test/parser_test.js

```diff
@@ -103,4 +103,23 @@ describe('parseLine', function () {
     var result = parser.parseLine(line);
     expect(result).to.deep.equal(expected);
   });
+  
+  it('should return correct values with headers', function () {
+    var line =  "Jan 13 10:21:38 localhost haproxy[29285]: 78.54.251.74:53867 [13/Jan/2015:10:20:36.708] www-https~ node-servers/node1-10.28.183.88 60988/0/2/384/61374 200 168 - - ---- 10/9/2/0/0 0/0 {HEADER} \"POST /api/1/traffic HTTP/1.1\"";
+    var expected = {
+      date: new Date('Jan 13 10:21:38').getTime(),
+      statusCode: "200",
+      haproxy: "haproxy.29285",
+      nodeserver: "node-servers/node1-10.28.183.88",
+      frontendConnections: "9",
+      backendConnections: "2",
+      backend: "node-servers",
+      backendServer: "node1",
+      totalTime: "61374",
+      totalRequestTime: "60988",
+      totalResponseTime: "384"
+    };
```

Comment:
lol. Thank you. I did not change the code, assumed we did but we did not.
Will fix now.